brmo-service/src/main/java/nl/b3p/brmo/service/scanner/GDS2OphalenProces.java

```diff
@@ -622,13 +622,30 @@ private Bericht laadAfgifte(AfgifteGBType a, String url) throws Exception {
             e.printStackTrace(new PrintWriter(sw));
             b.setOpmerking("Fout bij parsen BRK bericht: " + sw.toString());
         }
-
-        Stripersist.getEntityManager().persist(lp);
-        Stripersist.getEntityManager().persist(b);
-        Stripersist.getEntityManager().merge(this.config);
-        Stripersist.getEntityManager().flush();
-        Stripersist.getEntityManager().getTransaction().commit();
-        Stripersist.getEntityManager().clear();
+        try {
+            em.persist(lp);
+            em.persist(b);
+            em.merge(this.config);
+            em.flush();
+            em.getTransaction().commit();
+            em.clear();
+        }catch(PersistenceException pe){
+            if(!em.getTransaction().isActive()){
+                em.getTransaction().begin();
+            }
+            em.getTransaction().rollback();
+            em.getTransaction().begin();
+            log.debug("Cannot save bericht/laadproces:", pe);
+            log.debug("Bericht: " + b.getObject_ref() + ":" + b.getBr_orgineel_xml());
+            lp.setStatus(LaadProces.STATUS.STAGING_DUPLICAAT);
```

Comment:
feitelijk is niet het laadproces duplicaat, maar het bericht uit het LP, mogelijk daarom beter `LaadProces.STATUS.STAGING_NOK` gebruiken (het bericht kan geen status krijgen omdat het niet bestaat want niet geladen kan worden)

Ik zou het duplicaat bericht in de opmerking van het LP stoppen, dan blijft dat bewaard (logfiles worden geroteerd), normaal gezien zijn de GD2 brk berichten niet erg groot, dus ik denk dat dat wel past (en het blijft tenslotte een uitzonderlijke situatie)